packages/react/src/components/CustomerContribution/PackageRuleContribution.tsx

```diff
@@ -300,10 +300,10 @@ const PackageRuleContributionInternal: React.FC<Props> = ({ source }) => {
           id = "rc-comment"
           label={
             <span>
-              Comments <i>- optional</i>
+              Comments
```

Comment:
Do we have/need validation here now that it's not optional? Or is it just a visual thing?

Reply:
Just dont want to highlight the optional here, we hope customer could provide a short description to explain the change, but still it wont block them to submit the replacement suggestion.